mysite/w2/alchemy_sigils.py

```diff
@@ -51,23 +52,9 @@ def getSigilSpeedAdviceGroup(practical_maxed: bool) -> AdviceGroup:
     mga_label = f"Multi Group A: {mga:.3f}x"
 
     # Multi Group B = Summoning Winner Bonuses
-    bd = session_data.account.summoning['BattleDetails']
-    player_matches_total = (
-        (bd['Green'][9]['RewardBaseValue'] * bd['Green'][9]['Defeated'])
-        + (bd['Yellow'][5]['RewardBaseValue'] * bd['Yellow'][5]['Defeated'])
-        + (bd['Blue'][5]['RewardBaseValue'] * bd['Blue'][5]['Defeated'])
-        + (bd['Purple'][7]['RewardBaseValue'] * bd['Purple'][7]['Defeated'])
-        + (bd['Cyan'][3]['RewardBaseValue'] * bd['Cyan'][3]['Defeated'])
-    )
-    matches_total = (
-        bd['Green'][9]['RewardBaseValue']
-        + bd['Yellow'][5]['RewardBaseValue']
-        + bd['Blue'][5]['RewardBaseValue']
-        + bd['Purple'][7]['RewardBaseValue']
-        + bd['Cyan'][3]['RewardBaseValue']
-    )
-    mgb = ValueToMulti(player_matches_total * session_data.account.summoning['WinnerBonusesMultiRest'])
-    mgb_label = f"Multi Group B: {mgb:.3f}x"
+    summoning = session_data.account.summoning
+    summoning_speed = summoning['Bonuses']['<x Sigil SPD']['Value']
```

Comment:
```suggestion
    mgb = summoning['Bonuses']['<x Sigil SPD']['Value']
```

to keep it consistent with the other multi groups